plugins/Login/templates/invitation.twig

```diff
@@ -1,48 +1,59 @@
+{% set isCenteredLogo = false %}
+{% set showImageDiv = true %}
+{% set imageToShow = 'plugins/Login/images/01_Create password.png' %}
+{% set colClassOverride = 'col l4 offset-l2 m12' %}
+{% set bodyClass = 'onBoarding' %}
 {% extends '@Login/loginLayout.twig' %}
 
 {% block loginContent %}
-    <div class="contentForm invitationForm">
-        <div class="card">
-            <div class="card-content">
-                <div class="card-title">
-                    {{ "Login_InvitationTitle"|translate }}
-                </div>
+    <div class="contentForm invitationForm" id="invitationForm">
+        <br>
+        <br>
```

Comment:
Can we gradually start using margins or padding instead of hardcoded line breaks or is there a benefit in `<br>`s over CSS? Especially here when we are also adjusting/adding new CSS. Same for paragraphs with an non-breakable space in them.

Reply:
@michalkleiner done :+1: